lib/sidekiq.rb

```diff
@@ -127,9 +129,13 @@ def self.logger=(log)
     Sidekiq::Logging.logger = log
   end
 
+  # When set, overrides Sidekiq.options[:global_poll_interval_average] and sets the
+  # average interval that this process will delay before checking for scheduled jobs
+  # or job retries that are ready to run.
+  #
   # See sidekiq/scheduled.rb for an in-depth explanation of this value
   def self.poll_interval=(interval)
```

Comment:
Oh sure, will do.

When I'm learning a new library, I like to see what the options are. Sidekiq has bits and pieces of advice throughout the wiki, but I struggled to find a single convenient reference that layed out all of my options ... until I opened the source and found `Sidekiq::DEFAULTS`. So `Sidekiq.options` actually feels natural to me.

If you prefer accessors, would you be interested in seeing a `Sidekiq.options = Sidekiq::Config` singleton at some later point?